Websocket proxy test fails on cloud

https://github.com/Qiskit/qiskit-ibm-runtime/runs/8064111382?check_suite_focus=true#step:5:8729

```
FAIL: test_websocket_proxy (test.integration.test_results.TestIntegrationResults) (service=<QiskitRuntimeService>)
Test connecting to websocket via proxy.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/qiskit-ibm-runtime/qiskit-ibm-runtime/test/decorators.py", line 69, in _wrapper
    func(self, *args, **kwargs)
  File "/home/runner/work/qiskit-ibm-runtime/qiskit-ibm-runtime/test/integration/test_results.py", line 233, in test_websocket_proxy
    self.assertTrue(callback_called)
AssertionError: False is not true
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -4,6 +4,6 @@ requests_ntlm>=1.1.0
 numpy>=1.13
 urllib3>=1.21.1
 python-dateutil>=2.8.0
-websocket-client>=1.0.1
+websocket-client<=1.3.3
 typing-extensions>=4.0.0
 ibm-platform-services>=0.22.6

--- a/setup.py
+++ b/setup.py
@@ -25,7 +25,7 @@ REQUIREMENTS = [
     "numpy>=1.13",
     "urllib3>=1.21.1",
     "python-dateutil>=2.8.0",
-    "websocket-client>=1.0.1",
+    "websocket-client<=1.3.3",
     "typing-extensions>=4.0.0",  # remove when support for Python 3.7 is dropped (use "from typing import" instead)
     "ibm-platform-services>=0.22.6",
 ]

